.travis.yml

```diff
@@ -172,7 +180,32 @@ jobs:
         - export SHELL=cmd
         - source scripts/setup-helics-ci-options.sh
         - choco install boost-msvc-12
-      
+    
+    - os: windows
+      name: "MSYS"
+      if: (branch = develop AND type IN (cron)) OR (branch = master AND type IN (push, pull_request)) OR (branch =~ /(msys)/) OR (commit_message =~ /(msys)/)
+      env:
+        - DISABLE_INTERFACES="Python,Java"
+        - SKIP_TEST_RUN=true
+        - CORE_CI_TEST=true
+        - DISABLE_EXAMPLES=true
```

Comment:
any particular reason to disable the examples?

Reply:
No, just similar settings to the MinGW build.